### ISSUE TITLE
Transaction Failed please make sure you are connected to the internet

### ISSUE THREAD

**Issue by XxEnigmaticxX** [hivewallet/hive-ios#48]
not sure what is going on, but anytime i try to send a tx i get the following message

"Transaction Failed please make sure you are connected to the internet"

my phone is connected to the internet so im not sure whats going on. 

also if i keep trying to send the tx, it will eventually say it was successful but the tx is not viewable on any block explorer. 
![image1](https://cloud.githubusercontent.com/assets/10783330/9256418/dce51a82-41b4-11e5-96a1-7299d92aa1b2.PNG)

**Comment by ghost** [comment]
Try this version:

http://hive-js.herokuapp.com

If it doesn't work for you, I recommend sweeping the private keys into another wallet using the tools at bip32.org

Right now our Litecoin support is pretty broken.

**Comment by XxEnigmaticxX** [comment]
thats my problem, i lost my passphrase when i moved from one apt to another. is there any update on the way? will i ever be able to access my coins? or have i essentially lost all my coins?

also here is what a successful tx looks like

![image1 1](https://cloud.githubusercontent.com/assets/10783330/9257402/f6f07416-41ba-11e5-8333-dea65bedb9ed.PNG)

i cant pull up that tx on any block chain. 

where is the private key stored on the iphone. i have mine jailbroken and i can access the files. if i can get my hands on the private key  or the wallet file i can then sweep that correct?

**Comment by ghost** [comment]
If you lost your passphrase you're in a bad situation. Anyway, that's a very good question. Let me see if I can find out. I'd definitely grab the entire Hive application folder ASAP, plus any WebKit cache/cookie stores you can find, for starters.

**Comment by XxEnigmaticxX** [comment]
i can grab the entire hive app folder. doing that now actually. where can i find the webkit cache/cookie stores on my iphone?

**Comment by ghost** [comment]
That I am not sure of (I am not an iOS developer). May need to do some extra research.

**Comment by XxEnigmaticxX** [comment]
i have the entire hive folder copied over to my desktop on my machine. looking to seeif i can find where the cocahe/cookie stores are in IOS.

**Comment by XxEnigmaticxX** [comment]
just a quick update. if i close the hive app and restart it the pending tx goes away completely. it seems like the tx never gets broadcasted, or if it is being broadcasted just not being picked up by any nodes

**Comment by XxEnigmaticxX** [comment]
last update. i am able to send ltc to the address specified by  the wallet and the tx is immediately recognized and it goes from unconfirmed to confirmed. so im not really sure what is going on here.

**Comment by XxEnigmaticxX** [comment]
woke up today and my coins moved from the hive wallet to my electrum wallet. everything seems to be working proper. not sure if yall did any thing but thank you.

**Comment by ghost** [comment]
Yeah, we didn't do that. It's just a result of the fee issue (works always as far as we know, but just takes FOREVER)

**Comment by ingria** [comment]
Same problem. Cannot send bitcoins, iOS 9.0, fresh installation from the appstore.

**Comment by ghost** [comment]
Hi guys,

Hive on iOS is basically dead. We don't have any developers with Macs anymore, so it's not really possible to keep it up-to-date or event o fix this problem. We're looking for another iOS maintainer, but until we do (and until we announce a new version!), you should be using something like Bread Wallet.

The good news is, your BIP32/BIP39 passphrases should be immediately portable.
